src/ddprof_cli.cc

```diff
@@ -233,6 +233,9 @@ int DDProfCLI::parse(int argc, const char *argv[]) {
   app.add_flag("--show_samples", show_samples,
                "Display captured samples as logs.\n")
       ->group("Debug options");
+  app.add_flag("--use_timestamps", use_timestamps,
+               "Put timestamps on sampled events.\n")
```

Comment:
This feature could mention timeline (as this will make more sense for a customer)
When timeline is supported, I guess you would expect this to just be a `--timeline` option ?